core/domain/user_services.py

```diff
@@ -1604,3 +1604,18 @@ def is_admin(user_id):
     if user_role == feconf.ROLE_ID_ADMIN:
         return True
     return False
+
+
+def is_topic_manager(user_id):
+    """Checks if a user with given user_id is a topic manager.
+
+    Args:
+        user_id: str. The unique ID of the user.
+
+    Returns:
+        bool. True if user is a topic manager, False otherwise.
```

Comment:
Cleaner: `bool. Whether the user is a topic manager.`

Reply:
Done.